tools/arch/PKGBUILD

```diff
@@ -0,0 +1,119 @@
+# Maintainer: Massimiliano Torromeo <massimiliano.torromeo@gmail.com>
+# Contributor: Iwan Timmer <irtimmer@gmail.com>
+# Contributor: Mark Constable <markc@renta.net>
+# Contributor: Anatol Pomozov <anatol.pomozov@gmail.com>
+
+pkgbase=cockpit
+pkgname=(cockpit cockpit-pcp cockpit-test)
+pkgver=VERSION
+pkgrel=1
+pkgdesc='A systemd web based user interface for Linux servers'
+arch=('x86_64')
+url='https://cockpit-project.org/'
+license=(LGPL)
+makedepends=(krb5 libssh accountsservice perl-json perl-locale-po json-glib glib-networking
+             git intltool gtk-doc gobject-introspection networkmanager libgsystem xmlto npm pcp)
+source=("SOURCE"
+        "cockpit.pam"
+        "cockpit-ws.sysuser.conf"
+        "cockpit-wsinstance.sysuser.conf")
+sha256sums=('SKIP'
+            'a979e236681c6a06906937cf0f012e976347af5d6d7e7ae04a11acb01cc2689d'
+            '1ad9dad75858264778bd94799b60c651f7cc1c7f7fa1c54622174303e639287a'
+            '46ee8ecad7bc97ba588ab9471dde76e41c00daf40658902425626c3a1938b438')
+
+prepare() {
+  cd cockpit-$pkgver
+  # TODO: disable buggy package-lock check
+  sed -r '/^cmd_make_package_lock_json\b/ a exit 0' -i tools/node-modules
+}
+
+build() {
+  cd cockpit-$pkgver
+  ./configure \
+    --prefix=/usr \
+    --sbindir=/usr/bin \
+    --libexecdir=/usr/lib/$pkgname/ \
+    --sysconfdir=/etc \
+    --localstatedir=/var \
+    --disable-dependency-tracking \
+    --disable-silent-rules \
+    --with-cockpit-user=cockpit-ws \
+    --with-cockpit-ws-instance-user=cockpit-wsinstance
+  make all
+}
+
+package_cockpit() {
+  depends=(krb5 libssh accountsservice perl-json perl-locale-po json-glib glib-networking)
+  backup=('etc/pam.d/cockpit')
+  optdepends=("cockpit-pcp: reading performance metrics"
+              "cockpit-podman: user interface for managing podman containers"
+              "cockpit-machines: user interface for managing virtual machines"
+              "udisks2: manage hard disks"
+              "networkmanager: manage network connections"
+              "packagekit: manage packages")
+
+  cd cockpit-$pkgver
+  make DESTDIR="$pkgdir" install
+  rm -rf "$pkgdir"/usr/{src,lib/firewalld}
+  install -Dm644 "$srcdir"/cockpit.pam "$pkgdir"/etc/pam.d/cockpit
+  install -Dm644 "$srcdir"/cockpit-ws.sysuser.conf "$pkgdir"/usr/lib/sysusers.d/cockpit-ws.conf
+  install -Dm644 "$srcdir"/cockpit-wsinstance.sysuser.conf "$pkgdir"/usr/lib/sysusers.d/cockpit-wsinstance.conf
+
+  echo "z /usr/lib/cockpit/cockpit-session - - cockpit-wsinstance -" >> "$pkgdir"/usr/lib/tmpfiles.d/cockpit-tempfiles.conf
```

Comment:
`ccockpit-ws` is indeed a sysuser so it's uid is not known upfront, so that's likely why it's like it. Shall we convert these comments into an upstream ticket?

Reply:
With my Debian and now Fedora packaging background this looks really strange -- but I am not familiar with Arch packaging guidelines/best practices -- perhaps that just is the sanctioned approach. So I'll leave that up to you.